packages/react/src/SearchControl/SearchControl.tsx

```diff
@@ -597,5 +601,5 @@ function getTotal(search: SearchRequest, lastResult: Bundle): number {
       (lastResult.entry?.length ?? 0) +
       (lastResult.link?.some((l) => l.relation === 'next') ? 1 : 0);
   }
-  return total;
+  return total ?? 0;
```

Comment:
Minor, but I think this is unnecessary?  `total` should always be a number at this point.

Reply:
VSCode red squigglies did me dirty - not sure why the flagged this